share/models/ingest.py

```diff
@@ -372,10 +380,10 @@ class RawDatum(models.Model):
     # The sha256 of the datum
     sha256 = models.TextField(validators=[validators.MaxLengthValidator(64)])
 
-    datestamp = models.DateTimeField(null=True, help_text=(
+    datestamp = models.DateTimeField(default=pendulum.now, help_text=(
```

Comment:
Should this just be `auto_now=True` or `auto_add_now=True`? It will take into account the system's timezone, or is that being ignored on purpose? `pendulum.now` defaults to UTC, correct?